apps/calendar/style/settings.css

```diff
@@ -99,18 +99,15 @@ not actually shown since the top header should be transparent.
   background-color: #eeeeee;
 }
 
-#settings .calendars label > .name {
+#settings .calendars label.name {
   display: block;
   overflow: hidden;
   white-space: nowrap;
   text-overflow: ellipsis;
   width: -moz-calc(100% - 2rem);
   height: 100%;
-}
-
-#settings .calendars label > span {
-  display: block;
-  float: left;
+  /* !important needed to override component scoped style to match legacy styles */
+  font-size: 1.7rem !important;
```

Comment:
also need to set the `color: #606060;` otherwise text will be darker than before.